### PR TITLE
Nyomi mackey; WIP

### PR DESCRIPTION
**Pull request by nyomilm**

### CODE REVIEW

File: css/index.css [hunk 1]
```diff
@@ -1,3 +1,305 @@
...
+    display: flex;
+  }
+
+
+.header .navbar a:hover {
+    text-decoration: underline;
+    color: black;
+}
+
+.header .navbar .a:first-child {
```
Comment (AsaOfDiamonds) [inline css/index.css:102]: is there a reason you are using .a:first-child?

File: css/index.css [hunk 2]
```diff
@@ -1,3 +1,305 @@
...
+    margin-left: 0;
+  }
+  
+.header .navbar a:last-child {
+    margin-right: 0;
```
Comment (AsaOfDiamonds) [inline css/index.css:107]: is there a reason you are using a:last-child?

File: css/index.css [hunk 3]
```diff
@@ -1,3 +1,305 @@
...
+
+/* Footer Styles */
+
+footer {
+    text-align: center;
+    margin: 40px 0 20px;
+}
+
+/* Services Styles */
+
```
Comment (AsaOfDiamonds) [inline css/index.css:204]: you have css styles for services but I do not see a services.html page. You should be making the HTML page then and only then doing css

**Review by AsaOfDiamonds — APPROVED** [review]
good job on your project, see my comments in your code.

File: css/index.css (hunk 1 above)
Comment (nyomilm) [reply css/index.css:102]: I thought I needed the command to be more specific?

File: css/index.css (hunk 3 above)
Comment (nyomilm) [reply css/index.css:204]: That copied over from the last great idea project. I only did the index page.

**Pull request merged**